src/pages/benefits/viewAllBenefit/ApplicationDetails.tsx

```
@@ -1,137 +1,289 @@
 const ApplicationDetails: React.FC = () => {
   const { id } = useParams<{ id: string }>();
-  const [applicantData, setApplicantData] = useState<ApplicantData[] | null>(
-    null
-  );
-  const [loading, setLoading] = useState(false);
-  const [documentData, setDocumentData] = useState<DocumentData[]>([]);
-  const [status, setStatus] = useState<any[]>([]);
+  const [applicantData, setApplicantData] = useState<ApplicantData[]>([]);
+  const [loading, setLoading] = useState(true); // Default to true to show loading initially
+  const [documents, setDocuments] = useState<Document[]>([]);
+  const [applicant, setApplicant] = useState<Record<string, any> | null>(null);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)